KomodoPlatform/atomicDEX-Desktop
# Remove Band Oracle Code

Local version of https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/2229 for quick/simple actions workflow
closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2031

To test:
- Basic smoke test: login, make a swap, make a transaction.
- Confirm price values of coins are still present and sane.

The band oracle code used to provide price data. It has since been replaced with other sources. We just need to confirm removing the related code has no negative impact on price calcs etc.
